tests/backends/base/test_base.py

```diff
@@ -211,6 +212,21 @@ def test_wrapper_connection_specific(self):
         self.assertEqual(connection.execute_wrappers, [])
         self.assertEqual(connections["other"].execute_wrappers, [])
 
+    @skipIf(connection.vendor == "sqlite", "can't see cursor's executed queries")
+    @skipIf(
+        connection.settings_dict.get("OPTIONS").get("server_side_binding") is True,
+        "no debug info for psycopg server-side binding",
+    )
+    def test_wrapper_debug(self):
```

Comment:
This is PostgreSQL specific test and should be moved to the `tests/backends/postgresql/`.

Reply:
I understand the reasoning behind the rest of the comments but not this one. Here is my thinking:

The patch fixes a postgresql specific bug. However, the underlying functionality, namely the functionality of including the wrapper to the sql query when inspecting the last executed query with a debug cursor, seems to be desirable for all databases. 

For now, no code amendments are needed for the rest of the databases, since they work as is. But, if someone were to change the last_executed_query code, say, for `mysql` and their changes stopped last_executed_query of including the wrapper, I think it would be nice that a test fails to let them know. Otherwise, we would get a similar ticket to this one, but now for `mysql`.

What do you think?

Reply:
We can try to keep it here.